File: C/1101-sequencia-de-numeros-e-soma.cpp
#include <stdio.h>
int main (){
	
	int A, B, C, D = 0;
	
	scanf ("%i %i", &A, &B);
	
	while (A > 0 && B > 0){
		
	if (A > B){
	
	for (C = B; C <= A; C++){
		
	D = D + C;
	
	printf ("%i ", C);	
		
	}	
	
	printf ("Sum=%i\n", D);
		
	}
	else if (A < B){
	
	for (C = A; C <= B; C++){
		
	D = D + C;
	
	printf ("%i ", C);	
		
	}	
	
	printf ("Sum=%i\n", D);
		
	}
	
	else if (A == B){
		
	printf ("%i %i Sum=%i", A, B, A+B);	
	
	}
	
	A = 0;
	B = 0;
	C = 0;
	D = 0;
		
	scanf ("%i %i", &A, &B);
			
	}
	
	return 0;
}

File: Python/1101-sequencia-de-numeros-e-soma.py
while True:
    valores = str(input()).split()
    valorUm = int(valores[0])
    valorDois = int(valores[1])
    soma = 0
    if valorUm <= 0 or valorDois <= 0:
        break
    else:
        if valorUm > valorDois:
            for i in range(valorDois, valorUm+1):
                print(f'{i} ', end='')
                soma += i
            print(f'Sum={soma}')
        else:
            for i in range(valorUm, valorDois+1):
                print(f'{i} ', end='')
                soma += i
            print(f'Sum={soma}')